src/staging/src/templates/site.js

```diff
@@ -67,15 +68,18 @@ export default function ParkTemplate({ data }) {
     ["facilityType.rank", "facilityType.facilityName"],
     ["asc"]
   )
+  const campingFacilities = 
+    activeFacilities.filter(
+      facility => facility.facilityType.facilityName.toLowerCase().includes("camping")
+    )
+  const nonCampingFacilities = 
+    activeFacilities.filter(
+      facility => !facility.facilityType.facilityName.toLowerCase().includes("camping")
+    )
```

Comment:
Mike - is there a better way to do this? I'm fine with it.

Reply:
This is the same query that was used for protectedAreas, so it's fine.  But it does look bad